### PR TITLE
Update for wagtail 5.0

### PR DESCRIPTION
**Pull request by nickmoreton**
## This updates the package for Wagtail 5.0 support.

Support ticket: https://torchbox.monday.com/boards/1124794299/pulses/1124795787

### CODE REVIEW

File: wagtail_transfer/static_src/chooser/PageChooserResult.js [hunk 1]
```diff
@@ -99,7 +99,7 @@ class PageChooserResult extends React.Component {
           href={page.meta.html_url}
           target="_blank"
           rel="noopener noreferrer"
-          className="status-tag primary"
+          className="w-status w-status--primary"
```
Comment (katdom13) [inline wagtail_transfer/static_src/chooser/PageChooserResult.js:102]: Thanks. I thought this was only for the status tag:
`{% status...`

But I guess you do need to search for the particular classname `status-tag`

**Review by katdom13 — APPROVED** [review]
This LGTM, thanks!

**Comment by josecruset** [comment]
Can this branch be merged into the main branch so it is usable for Wagtail 5.0/5.1? Thank you very much

**Comment by gasman** [comment]
Merged as part of #150.

**Pull request closed without merging**